src/Servers/HttpSys/src/HttpSysListener.cs

```diff
@@ -264,7 +264,7 @@ private void DisposeInternal()
         /// <summary>
         /// Accept a request from the incoming request queue.
         /// </summary>
-        public Task<RequestContext> AcceptAsync()
+        public Task<RequestInitalizationContext> AcceptAsync()
```

Comment:
ValueTask?

Reply:
Yes it is called in parallel.

Reply:
Not reusing, but using ValueTask which means can dump the TCS https://github.com/aspnet/AspNetCore/pull/10283/commits/fd0eab7ea2ee5aed8389d5adf29a57188e5536a5